Introduce multi host in pulsar doc (#4546)
apache/pulsar: site2/docs/client-libraries-java.md

pulsar://localhost:6650
```

> If you have more than one broker, the URL will look like something like this:

Reviewer: ```suggestion
If you have more than one broker, the URL may look like this:
```

        .build();
```

> If you have mutliple brokers, you can instantiate a PulsarClient like this:

Reviewer: ```suggestion
If you have multiple brokers, you can initiate a PulsarClient like this:
```
